fastlane/docs/AvailablePlugins.md

```diff
@@ -0,0 +1,5 @@
+### Available Plugins
+
+| Plugin Name | Description | Downloads
+--------------|-------------|----------|----------
+[apprepo](https://github.com/suculent/fastlane-plugin-apprepo) | experimental fastlane plugin based on https://github.com/suculent/apprepo SFTP uploader | 111
```

Comment:
Absolutely. I was thinking even doing this every day on some CI machine that will submit a PR. 
I'd still like the docs to be available on GitHub too, so people can see the available plugins without having to install and start using `fastlane`. 
Let's start with this, and see how it goes 👍